tests/fixtures/commits.json

```diff
@@ -1,5 +1,5 @@
+{"author": "Gijs Kruitbosch <gijskruitbosch@gmail.com>", "desc": "Bug 1388990 - avoid breaking the broadcast/listener relationships of the back/fwd buttons when customizing, r=dao\n\nMozReview-Commit-ID: EsH3lv8QXfJ", "date": "2017-08-12 10:56:22", "pushdate": "2017-08-12 21:35:10", "bug_id": 1388990, "ever_backedout": false, "added": 9, "test_added": 38, "deleted": 2, "test_deleted": 0, "files_modified_num": 4, "types": [".xul", "JavaScript", ".ini"], "components": ["Firefox::Toolbars and Customization", "Firefox::General"], "author_experience": 288, "author_experience_90_days": 117, "author_email": "gijskruitbosch@gmail.com", "node": "9d576871fd33bed006dcdccfba880a4ed591f870"}
+{"author": "Steve Chung <schung@mozilla.com>", "desc": "Bug 1390433 - Uplift Form Autofill system add-on for 56 beta. r=MattN\n\nMozReview-Commit-ID: LX0gELG2oUC", "date": "2017-08-25 16:57:39", "pushdate": "2017-08-25 23:22:16", "bug_id": 1390433, "ever_backedout": false, "added": 8, "test_added": 0, "deleted": 3, "test_deleted": 0, "files_modified_num": 2, "types": [".build", ".in"], "components": ["Firefox Build System::General"], "author_experience": 10, "author_experience_90_days": 0, "author_email": "schung@mozilla.com", "node": "3d38ebe3179eff291055e3f49f49479e9ff75504"}
+{"author": "Kearwood \"Kip\" Gilbert <kgilbert@mozilla.com>", "desc": "Bug 1389220 - Prevent navigator.activeVRDisplays from activating VR hardware and runtimes r=daoshengmu\n\nMozReview-Commit-ID: JYMiGyT0iVY", "date": "2017-09-20 22:39:35", "pushdate": "2017-09-21 13:14:27", "bug_id": 1389220, "ever_backedout": false, "added": 8, "test_added": 0, "deleted": 3, "test_deleted": 0, "files_modified_num": 1, "types": ["C/C++"], "components": ["Core::DOM: Core & HTML"], "author_experience": 7, "author_experience_90_days": 7, "author_email": "kgilbert@mozilla.com", "node": "b37909c3b506a2196d9e7c41c8f72e23b15eb3c9"}
+{"author": "Mike Hommey <mh+mozilla@glandium.org>", "desc": "Bug 1488307 - Build the LLVM gold plugin. r=mshal\n\nAs much as it's tempting to use LLD for LTO, it still causes some\nsubtle problems with the build, and it's still better to keep using\nBFD ld for the time being. Doing so requires the gold plugin, which\nonly requires to pass cmake the directory where the binutils headers\nare, and they are part of the gcc toolchain headers.\n\nDifferential Revision: https://phabricator.services.mozilla.com/D4896", "date": "2018-08-30 01:55:49", "pushdate": "2018-09-05 09:45:59", "bug_id": 1488307, "ever_backedout": true, "added": 2, "test_added": 0, "deleted": 0, "test_deleted": 0, "files_modified_num": 1, "types": ["Python"], "components": ["Firefox Build System::General"], "author_experience": 1147, "author_experience_90_days": 125, "author_email": "mh+mozilla@glandium.org", "node": "e2a02b08089b0bd0c18ceac0b2eb1e3888d56dc2"}
```

Comment:
@marco-c these last two commits have their "ever_backedout" values wrongly swapped, can that be a problem?

Reply:
What do you mean by "wrongly swapped"?

Reply:
@marco-c means the one which has "ever_backedout" =True is actually False in data/commits.json , and vice-versa

Reply:
Ah OK, it shouldn't be a problem, but better to fix it.